docs/source/receptorctl/receptorctl_work_submit.rst

```diff
@@ -0,0 +1,39 @@
+-----------
+work submit
+-----------
+
+.. contents::
+   :local:
+
+``receptorctl work submit`` requests a Receptor node to run a unit of work.
+
+Command syntax: ``receptorctl --socket=<socket_path> work submit [<<Options>>] <<WorkType>> [<<Command Parameters>>]``
+
+``socket_path`` is the control socket address for the Receptor connection.
+   The default is ``unix:`` for a Unix socket.
+   Use ``tcp://`` for a TCP socket.
+   The corresponding environment variable is ``RECEPTORCTL_SOCKET``.
+
+.. code-block:: text
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <pid>``
+
+``WorkType`` is the execution request type for the work unit.
+
+``Command Parameters`` are parameters passed to the work command.
```

Comment:
```suggestion
``Command Parameters`` modify the requests that you send with the ``work`` command. The following are the available parameters:
```

Reply:
Not sure what happened but my previous suggestion seems to have been botched. Hopefully this new suggestion is good. I would def recommend introducing the list of parameters with something like "the following:" or "as follows:"

Reply:
Please see https://ansible.readthedocs.io/projects/receptor/en/latest/user_guide/workceptor.html#runtime-parameters for how Command Parameters work.
They are distinct from the "Options" listed below.  The terms are confusingly similar and we may want to distinguish between Receptor-related parameters/options and worktype parameters.

Reply:
Can we change ``Command Parameters`` to ``Runtime Parameters`` here?